packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -25,9 +25,28 @@ const farmExpenseTypeController = {
       try {
         const data = req.body;
         data.expense_translation_key = data.expense_name;
-        const result = await baseController.postWithResponse(ExpenseTypeModel, data, req, { trx });
-        await trx.commit();
-        res.status(201).send(result);
+
+        // check if records exixts in DB
+        const record = await ExpenseTypeModel.query()
+          .where({
+            expense_name: data.expense_name,
+            farm_id: data.farm_id,
+          })
+          .first();
+
+        //if record exists then set 'deleted' column to false
+        if (record) {
```

Comment:
Should this be `if (record && record['deleted'] == true)` ?

Else we are changing updated_at in the database if it already exists and is already false.